config/runtime-template.yml

```diff
@@ -10,6 +10,10 @@ prometheusConfig:
   exposeMetrics: true
 slackConfig:
   alertUrl: # required for slack integration to work
+tokenOAuthConfig:
+  ClientID: "example-client"
+  ClientSecret: "example-client-secret"
```

Comment:
I think we might want to pass these parameters in via a separate config file because when we deploy this on k8s, this config is a  `ConfigMap`. The main config in this template could be a path to a file with the client id and client secret in it.

Reply:
You are right. ClientId is public but clientSecret should not be. If you are interested in this change to go in, I can raise a new PR since this one got merged. Let me know.

Reply:
I created https://github.com/datastax/pulsar-heartbeat/issues/32 to track this additional feature. It'd be very helpful if you can contribute this additional feature, just let me know, thanks!